smother/control.py

```diff
@@ -41,7 +46,7 @@ def start(self):
 
     def save_context(self, label):
         self.data[label] = {
-            key: sorted(map(int, val.keys()))
+            self._normalize_path(key): sorted(map(int, val.keys()))
```

Comment:
> Relative paths are kind of nice if you're manually looking at the file, but not required for portability since you can always achieve the same thing with a .coveragerc and smother combine. I don't think that's a strong enough reason to add a new flag to smother.

This is only true if we make our own PathAliases that doesn't use the canonical path. Otherewise there is no way (without editing the file ourselves) to get relative paths. We could do that, but that might mess with people's assumptions that it has the same behavior as coverage. What if we have a keep the `--relative-paths` option only for `combine` as a nice bonus feature?